src/main/webapp/product.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Product Listing</title>
+    <link rel="stylesheet" href="assets/css/product-page.css" />
+    <link rel="stylesheet" href="assets/css/lightbox.css" />
+    <script src="assets/js/lightbox.js"></script>
```

Comment:
The directory structure is bugged! I think you meant to put js in the same level as assets, not under css? When I ran the server, it showed Failed to load resource.
![Screenshot_1](https://user-images.githubusercontent.com/50647023/86689142-4fa36a80-c039-11ea-8b02-4d2178654e2d.jpg)

Reply:
oh yes thank you! no wonder why my js wasn't working! i thought i would debug it proper when cloud shell is up (it's been down). I'll make changes